sortinghat/core/api.py

```diff
@@ -607,7 +607,7 @@ def delete_domain(ctx, domain_name):
 
 
 @django.db.transaction.atomic
-def enroll(ctx, uuid, organization, from_date=None, to_date=None):
+def enroll(ctx, uuid, organization, from_date=None, to_date=None, ignore_default=False):
```

Comment:
I think we should rename `ignore_default` to `force` or something like that to overwrite default ranges. It's shorter and easier to remember - specially for the command line arguments.

Reply:
Ok, I have renamed it also in the mutation.